kie-dmn-feel/src/test/java/org/kie/dmn/feel/runtime/FEELStaticTypeTest.java

```diff
@@ -112,6 +112,17 @@
                   }},
                   new HashMap<String,Object>() {{
                       put( "myFeelVar", "John zip code is: 12345" );
+                  }} },
+                
+                { "{ myFeelVar : person.first name + \"home street name is: \" + person.address.street name }",
```

Comment:
@etirelli I've realized a FEEL expression as `person.address.street name` would have triggered a "no viable alternative..." error.

With this PR I fix this bug, other misc improvements (resolving as symbol also standard JavaBean properties) and I introduced the `@FEELType` annotation.